pkg/work/spoke/controllers/manifestcontroller/manifestwork_reconciler.go

```
@@ -241,19 +242,31 @@ func allInCondition(conditionType string, manifests []workapiv1.ManifestConditio
 
 func buildAppliedStatusCondition(result applyResult, generation int64) metav1.Condition {
 	if result.Error != nil {
+		// Check if this is an ignoreFields processing error
+		reason := workapiv1.AppliedManifestFailed
+		message := fmt.Sprintf("Failed to apply manifest: %v", result.Error)
+		
```

golangci-lint (2.5.0)

[error] File is not properly formatted

(gci)